plugins/rbac-tool.yaml

```
@@ -0,0 +1,46 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: rbac-tool
+spec:
+  version: v1.1.1
+  platforms:
+  - bin: rbac-tool
+    uri: https://github.com/alcideio/rbac-tool/releases/download/v1.1.1/rbac-tool_v1.1.1_linux_amd64.tar.gz
+    sha256: ecdc8b365b8f9bb4303d194e777a9e7fdf3376158e3a2fb78cf7425007118a1d
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+
+  - bin: rbac-tool
+    uri: https://github.com/alcideio/rbac-tool/releases/download/v1.1.1/rbac-tool_v1.1.1_darwin_amd64.tar.gz
+    sha256: 038d583a98783bf72324eda1c9bab88edff115c1a0b03f4e67c27ea55f7a6407
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+
+  - bin: rbac-tool.exe
+    uri: https://github.com/alcideio/rbac-tool/releases/download/v1.1.1/rbac-tool_v1.1.1_windows_amd64.tar.gz
+    sha256: 1cd62b006b1cfc484866760ca846ca3d2701791e300a1f419b756e29cdc7b6a8
+    selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+
+  shortDescription: RBAC power toys to analyze permissions and generate policies
```

```suggestion
  shortDescription: Plugin to analyze RBAC permissions and generate policies
```

I'd stay away from "power tools", because it is subjective.

---

Noted and fixed